Homework7/lesson6/my_weather.py

```diff
@@ -0,0 +1,39 @@
+from requests import get
+import json
+import sys
+
+nargs = len(sys.argv)
+#print(nargs)
```

Comment:
Please remove comment

Reply:
Fixed. Number of args used as a משתנה עזר .